pkg/controller/storagecluster/uninstall_reconciler_test.go

```diff
@@ -131,3 +137,78 @@ func TestDeleteStorageClasses(t *testing.T) {
 		}
 	}
 }
+
+func TestDeleteNodeAffinityKeyFromNodes(t *testing.T) {
```

Comment:
use Table Driven approach and do processing steps  in separate functions outside the main test block for better readability.

Reply:
Done